ui/components/component-library/badge-wrapper/badge-wrapper.scss

```diff
@@ -32,8 +32,8 @@
 }
 
 .mm-badge-wrapper {
-  --badge-wrapper-position-circular: 14%;
-  --badge-wrapper-position-rectangular: 0;
+  --badge-wrapper-position-circular: 7%;
+  --badge-wrapper-position-rectangular: 11%;
```

Comment:
non-blocking: Not sure if this will affect all use cases. Here’s an audit I did on the `BadgeWrapper` component:  

https://docs.google.com/spreadsheets/d/1W6Pfd20gdjaalQ2hSBbA9slRLJUXmE5fwQ7WIhvt7_0/edit?gid=1856597125#gid=1856597125  

We may need to visually check each instance that isn’t using the `AvatarNetwork` component to be safe.

Reply:
Thanks for checking! Also double checked that the other usages look right